### PR TITLE
glibmm: Edit m4 full path reference

### PR DESCRIPTION
**Pull request by Biswa96**

### CODE REVIEW

File: mingw-w64-glibmm/PKGBUILD [hunk 1]
```diff
@@ -68,6 +68,9 @@ package() {
 
   DESTDIR="${pkgdir}" ${MINGW_PREFIX}/bin/meson.exe install
 
+  local _BINDIR=$(cygpath -wm /usr/bin)
+  sed -s "s|${_BINDIR}|/usr/bin|g" -i "${pkgdir}${MINGW_PREFIX}"/lib/glibmm-2.4/proc/gmmproc
```
Comment (lazka) [inline mingw-w64-glibmm/PKGBUILD:72]: Should the shebang of that file also be switched to /usr/bin/perl?

Comment (Biswa96) [reply mingw-w64-glibmm/PKGBUILD:72]: Yup.

Comment (Biswa96) [reply mingw-w64-glibmm/PKGBUILD:72]: Feel free to take it over.

Comment (lazka) [reply mingw-w64-glibmm/PKGBUILD:72]: ok :)

**Pull request closed without merging**